apps/andi/test/support/auth_helper.ex

```diff
@@ -34,6 +34,7 @@ defmodule Andi.Test.AuthHelper do
       |> Plug.Session.call(signing_opts)
       |> Plug.Conn.fetch_session()
       |> TokenHandler.put_session_token(jwt)
+      |> Plug.Conn.put_session(:user_id, "TEST")
```

Comment:
This here in particular highlights why this approach is not great. Any use could set their user_id to ANY value and we have no mechanisms in place to validate that. If it all stems from the JWT, then we can verify the signature, expiry, etc.

Reply:
LiveView does make this tricky with its use of session, so we may have to lock that down as well.